Tests/SwiftPrettyPrintTests/Public/CombineExtensionTests.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  CombineExtensionTests.swift
+//  SwiftPrettyPrintTests
+//
+//  Created by Yusuke Hosonuma on 2020/11/26.
+//
+
+// Linux is not supported to Combine framework.
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+
+import XCTest
+import Combine
+@testable import SwiftPrettyPrint
+
+// Note:
+// Original `print` operator output is:
+// https://developer.apple.com/documentation/combine/publisher/print(_:to:)
+//
+// ```
+// receive subscription: ([[1, 2], [3, 4]])
+// request unlimited
+// receive value: ([1, 2])
+// receive value: ([3, 4])
```

Comment:
Default implementation is enclose in parentheses but remove it in our implementation because redundant.